src/hmem.c

```diff
@@ -359,3 +359,13 @@ size_t ofi_hmem_get_ipc_handle_size(enum fi_hmem_iface iface)
 		fi_strerror(-ret));
 	return 0;
 }
+
+int ofi_num_hmem_interfaces_initialized(void) {
+	int i, hmem_init_count = 0;
+	for (i = 1; i < FI_NUM_HMEM_TYPES; i++) {
+		if (hmem_ops[i].initialized == true) {
+			hmem_init_count++;
+		}
+	}
+	return hmem_init_count;
```

Comment:
This would be another place to use the iterator defined in this PR:
https://github.com/ofiwg/libfabric/pull/8351/files#diff-c8ef9d9845e5822a14e1c219002ba1c60a2b44c4ab90a321cf34321882b41eb6

Reply:
Also, assuming we're happy with this function, it should be renamed to indicate it's counting the number of **non-standard** `FI_HMEM` interfaces, since the loop starts at 1.

Reply:
fixed.